azure-cognitiveservices-vision-computervision/README.md

```diff
@@ -0,0 +1,327 @@
+
+```Bash
+python3 -m venv cogsrv-vision-env
+source cogsrv-vision-env/bin/activate
+```
+
+### Install the SDK
+
+Install the Azure Cognitive Services Computer Vision SDK for Python with [pip][pip]:
+
+```Bash
+pip install azure-cognitiveservices-vision-computervision
+```
+
+## Authentication
+
+Once you create your Computer Vision resource, you need its **region**, and one of its **account keys** to instantiate the client object.
+
+Use these values when you create the instance of the [ComputerVisionAPI][ref_computervisionclient] client object. 
+
+### Get credentials
+
+Use the [Azure CLI][cloud_shell] snippet below to populate two environment variables with the Computer Vision account **region** and one of its **keys** (you can also find these values in the [Azure portal][azure_portal]). The snippet is formatted for the Bash shell.
+
+```Bash
+RES_GROUP=<resourcegroup-name>
+ACCT_NAME=<computervision-account-name>
+
+export ACCOUNT_REGION=$(az cognitiveservices account show \
+    --resource-group $RES_GROUP \
+    --name $ACCT_NAME \
+    --query location \
+    --output tsv)
+
+export ACCOUNT_KEY=$(az cognitiveservices account keys list \
+    --name $ACCT_NAME \
+    --resource-group $RES_GROUP \
+    --name $ACCT_NAME \
+    --query key1 \
+    --output tsv)
+```
+
+### Create client
+
+Once you've populated the `ACCOUNT_REGION` and `ACCOUNT_KEY` environment variables, you can create the [ComputerVisionAPI][ref_computervisionclient] client object.
+
+```Python
+from azure.cognitiveservices.vision.computervision import ComputerVisionAPI
+from azure.cognitiveservices.vision.computervision.models import VisualFeatureTypes
+from msrest.authentication import CognitiveServicesCredentials
+
+import os
+region = os.environ['ACCOUNT_REGION']
+key = os.environ['ACCOUNT_KEY']
+
+credentials = CognitiveServicesCredentials(key)
+client = ComputerVisionAPI(region, credentials)
+```
+
+## Usage
+
+Once you've initialized a [ComputerVisionAPI][ref_computervisionclient] client object, you can:
+
+* Analyze an image: You can analyze an image for certain features such as faces, colors, tags.   
+* Generate thumbnails: Create a custom JPEG image to use as a thumbnail of the original image.
+* Get description of an image: Get a description of the image based on its subject domain. 
+
+For more information about this service, see [What is Computer Vision?][computervision_docs].
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Computer Vision tasks, including:
+
+* [Analyze an image](#analyze-an-image)
+* [Analyze an image by domain](#analyze-an-image-by-domain)
+* [Get text description of an image](#get-text-description-of-an-image)
+* [Get handwritten text from image](#get-text-from-image)
+* [Generate thumbnail](#generate-thumbnail)
+* [Get subject domain list](#get-subject-domain-list)
```

Comment:
Would be good to bump [Get subject domain list](#get-subject-domain-list) up under [Analyze an image](#analyze-an-image) to reflect the section move.